lib/src/infrastructure/ucids_tokens.repository.dart

```diff
@@ -1,8 +1,8 @@
 /// SPDX-License-Identifier: AGPL-3.0-or-later
 
 import 'dart:convert';
-import 'package:archethic_dapp_framework_flutter/src/domain/models/ucids_tokens.dart';
-import 'package:archethic_dapp_framework_flutter/src/domain/repositories/tokens/ucids_tokens.repository.dart';
+
+import 'package:archethic_dapp_framework_flutter/archethic_dapp_framework_flutter.dart';
```

Comment:
is it better to import the class with export definition or directly the src dart file ?

Reply:
Fixed 👍